### ISSUE TITLE
Drew soma, hidden in the corner.

### ISSUE THREAD

**Issue by eleftherioszisis** [BlueBrain/NeuroM#615]
Well, there is something amiss with the centering of the drawn soma...

``` python
neuron = load_neuron('Neuron.swc')
viewer.draw(neuron.soma)
```
![image](https://cloud.githubusercontent.com/assets/15011025/24085699/72dc1da0-0cf8-11e7-9cdd-f2667e5ef29e.png)

**Comment by mgeplf** [comment]
Which version are you using?
How do you expect it to look?

**Comment by mgeplf** [comment]
I think you're using the last release, and not the latest git version.  I refactored a lot of the drawing code, and the new behavior will put the soma in the center of the figure.  Can you try the latest version?

**Comment by eleftherioszisis** [comment]
Well, I used pip install --upgrade to install NeuroM.
The issue is that I create a tutorial for Felix's course and the students will not git clone NeuroM.

**Comment by mgeplf** [comment]
I can release a new version, but I want to make sure that solves your problem first.

**Comment by eleftherioszisis** [comment]
I tried it with the new git version and it is indeed fixed, albeit it is tiny in the limits of the figure.

What is most important for the new version though is to be able to load morphologies from neuromorpho, which I cannot do know and I tried with many different cells including the Markram lab.
Therefore, the correct location of the soma is of low priority.

**Comment by mgeplf** [comment]
The default white_space is 30, which was the original default.  The Neuron.swc file has sizes in the 0.1 range, so if you set white_space=1, it will display better.

**Comment by mgeplf** [comment]
Already fixed in #608
